features.js

```diff
@@ -800,21 +800,27 @@ module.exports = {
 
     // check whether video is received after 10 seconds
     receivingvideo10s: function(client, peerConnectionLog) {
-        var count = 0;
         var receivedVideo = undefined;
+        var timeConnected = false;
         for (var i = 0; i < peerConnectionLog.length; i++) {
-            if (peerConnectionLog[i].type === 'getStats') {
-                if (count++ === 10) {
-                    Object.keys(peerConnectionLog[i].value).forEach(id => {
-                        var report = peerConnectionLog[i].value[id];
-                        if (report.type === 'ssrc' && (report.kind === 'video' || report.mediaType === 'video') && id.indexOf('_recv')) {
-                            receivedVideo = {
-                                packetsReceived: report.packetsReceived,
-                                frameWidth: report.frameWidth,
-                                frameHeight: report.frameHeight
-                            };
-                        }
-                    });
+            if (peerConnectionLog[i].type === 'oniceconnectionstatechange' && !timeConnected) {
+                if (peerConnectionLog[i].value === 'connected' || peerConnectionLog[i].value === 'completed') {
```

Comment:
I find this approach more readable
```
const iceState = peerConnectionLog[i].value;
if (['connected', 'completed'].includes(iceState)]) {
```

Reply:
that is one of the things enabled by es6. But remember this code was written when ES6 was not yet en vogue.

I'd rather avoid introducing too much es6 now and rather do a single bit "port it to es6" some day.